omnibot/processor.py

```diff
@@ -370,7 +370,6 @@ def _handle_message_callback(message, callback):
             message.match_type,
             message.match
         ),
-        extra={**message.event_trace, 'callback': callback},
```

Comment:
ah yes just callback, let me add event trace back in

Reply:
Can we keep some of the data provided by the callback config?
I believe it would be beneficial for debugging purposes to keep the callback module name logged, as well as potentially the request path.

The callback dict typically looks something like :
```json
      "callback":{
         "kwargs":{
            "client_kwargs":{
               "service":"service_name",
               ... other client args (potentially sensitive data) ...
            }, 
            "request_kwargs":{
               "path":"/api/path/to/other/service/endpoint"
               ... other request args ...
            }
         },
         "module":"python.callback.module.name"
      }
```

Reply:
@kvnxiao thanks, addressed